migration_steps/validation/validate_db/app/letters/sirius_dates.json

```diff
@@ -0,0 +1 @@
+{"2022-01-21":{"bs1_from":"2022-01-18","bs1_to":"2022-01-18","bs2_from":"2021-12-29","bs2_to":"2021-12-29","rd1_from":"2022-02-11","rd1_to":"2022-02-11","rd2_from":"2022-01-20","rd2_to":"2022-01-20","ri2_from":"2022-01-18","ri2_to":"2022-01-18","ri3_from":"2021-12-29","ri3_to":"2021-12-29","rr1_from":"2022-01-18","rr1_to":"2022-01-18","rr2_from_lay":"2021-12-06","rr2_to_lay":"2021-12-06","rr2_from_pro_pa":"2021-10-25","rr2_to_pro_pa":"2021-10-25","rr3_from":"2022-01-18","rr3_to":"2022-01-18"},"2022-01-22":{"bs1_from":"2022-01-19","bs1_to":"2022-01-19","bs2_from":"2021-12-30","bs2_to":"2021-12-30","rd1_from":"2022-02-12","rd1_to":"2022-02-12","rd2_from":"2022-01-21","rd2_to":"2022-01-21","ri2_from":"2022-01-19","ri2_to":"2022-01-19","ri3_from":"2021-12-30","ri3_to":"2021-12-30","rr1_from":"2022-01-19","rr1_to":"2022-01-19","rr2_from_lay":"2021-12-07","rr2_to_lay":"2021-12-07","rr2_from_pro_pa":"2021-10-26","rr2_to_pro_pa":"2021-10-26","rr3_from":"2022-01-19","rr3_to":"2022-01-19"},"2022-01-23":{"bs1_from":"2022-01-19","bs1_to":"2022-01-19","bs2_from":"2021-12-30","bs2_to":"2021-12-30","rd1_from":"2022-02-13","rd1_to":"2022-02-13","rd2_from":"2022-01-22","rd2_to":"2022-01-22","ri2_from":"2022-01-19","ri2_to":"2022-01-19","ri3_from":"2021-12-30","ri3_to":"2021-12-30","rr1_from":"2022-01-19","rr1_to":"2022-01-19","rr2_from_lay":"2021-12-07","rr2_to_lay":"2021-12-07","rr2_from_pro_pa":"2021-10-26","rr2_to_pro_pa":"2021-10-26","rr3_from":"2022-01-19","rr3_to":"2022-01-19"},"2022-01-24":{"bs1_from":"2022-01-19","bs1_to":"2022-01-19","bs2_from":"2021-12-30","bs2_to":"2021-12-30","rd1_from":"2022-02-14","rd1_to":"2022-02-14","rd2_from":"2022-01-23","rd2_to":"2022-01-23","ri2_from":"2022-01-19","ri2_to":"2022-01-19","ri3_from":"2021-12-30","ri3_to":"2021-12-30","rr1_from":"2022-01-19","rr1_to":"2022-01-19","rr2_from_lay":"2021-12-07","rr2_to_lay":"2021-12-07","rr2_from_pro_pa":"2021-10-26","rr2_to_pro_pa":"2021-10-26","rr3_from":"2022-01-19","rr3_to":"2022-01-19"},"2022-01-25":{"bs1_from":"2022-01-20","bs1_to":"2022-01-20","bs2_from":"2021-12-31","bs2_to":"2021-12-31","rd1_from":"2022-02-15","rd1_to":"2022-02-15","rd2_from":"2022-01-24","rd2_to":"2022-01-24","ri2_from":"2022-01-20","ri2_to":"2022-01-20","ri3_from":"2021-12-31","ri3_to":"2021-12-31","rr1_from":"2022-01-20","rr1_to":"2022-01-20","rr2_from_lay":"2021-12-08","rr2_to_lay":"2021-12-08","rr2_from_pro_pa":"2021-10-27","rr2_to_pro_pa":"2021-10-27","rr3_from":"2022-01-20","rr3_to":"2022-01-20"}}
```

Comment:
What's the purpose of this file, just for my information?

Reply:
I wrote a little script in Sirius that takes a date as input and spits out this json of variables (dates) for the letter selection SQL.

For example, if today is `2022-01-21`, the date range for BS1 letters will be from `2022-01-18` to `2022-01-18`.

Basically didn't want to rewrite the date calculation logic in SQL, when I only need to test a few specific dates